README.md

```diff
@@ -4,6 +4,58 @@ Enables Postman support of the WSDL specification
 browserify index.js --standalone xsd2jsonschemafaker > xsd2jsonschemafaker.js
 
 
+## Command Line Interface
+
+The converter can be used as a CLI tool as well. The following [command line options](#options) are available.
+
+`wsdl2postman [options]`
+
+### Options
+- `-v`, `--version`  
+  Specifies the version of the converter
+
+- `-s <source>`, `--spec <source>`  
+  Used to specify the WSDL specification (file path) which is to be converted
+
+- `-o <destination>`, `--output <destination>`  
+  Used to specify the destination file in which the collection is to be written
+
+- `-t`, `--test`  
+  Used to test the collection with an in-built sample specification
+
+- `-p`, `--pretty`  
+  Used to pretty print the collection object while writing to a file
+
+- `-O`, `--options`
+  Used to supply options to the converter
+
+- `-c`, `--options-config`  
+  Used to supply options to the converter through config file
+
+- `-h`, `--help`  
+  Specifies all the options along with a few usage examples on the terminal
+
+
+### Usage
+
+**Sample usage examples of the converter CLI**
+
+
+- Takes a specification (spec.wsdl) as an input and writes to a file (collection.json) with pretty printing and using provided options
```

Comment:
Can we also create an `OPTIONS.md` file that mentions the usage of options? Basically, this file should display what we already have under `lib/utils/options.js` in a table. For example, [OPTIONS.md](https://github.com/postmanlabs/openapi-to-postman/blob/develop/OPTIONS.md) in openapi-to-postman module.

Also the script that generates documentation is here: https://github.com/postmanlabs/openapi-to-postman/blob/develop/test/system/structure.test.js#L174

Reply:
Added in last commits